graphene_django/relay/connection.py

```diff
@@ -305,3 +294,18 @@ def _handle_last_before(
         has_previous_page,
         has_next_page,
     )
+
+
+def _use_connection_dataloaders(info, sized_sliceable):
+    if (
```

Comment:
why not simply something like:
```python
return (
    django.VERSION[0] >= 3
    and graphene_settings.USE_DATALOADERS
    ...
)
```

and remove the if statement

Reply:
LOL too late @tcleonard